5.1.thats_the_way.py

```diff
@@ -0,0 +1,12 @@
+import fnmatch
+import os
+
+
+def deepfiles(path):
+    """
+     Return a list of files in `path` starting with "deep".
+     """
+    return fnmatch.filter(os.listdir(path), 'deep*')
+
+
+print(deepfiles('.\\images'))
```

Comment:
Expressions outside of a function are a bad habit. This code will be called every time the file is loaded and imported.   Read more here: https://www.geeksforgeeks.org/python-main-function/amp/